exopy_pulses/pulses/shapes/gaussian_shape.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2021 by ExopyPulses Authors, see AUTHORS for more details.
```

Comment:
I am always a bit unclear on copyright stuff but I believe the copyright cannot predate the file existence so it would be just 2021 for this one.